docs/intro-to-eda.md

```diff
@@ -1,56 +1,53 @@
 ---
-title: Learn About Event Driven Architecture
+title: About Event Driven Architecture at VA
 ---
 
 # Introduction
-The Enterprise Event Bus is an asynchronous event processing system that spans systems and lines of business at the VA. Event-driven architecture uses events, which are specific instances of something that happened, to communicate with systems that are subscribed to the stream of events. The systems that are producing the events [producers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/produce-events/), are decoupled from the systems that are consuming their events [consumers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/consume-events/).
+The Enterprise Event Bus is an asynchronous event processing system that spans systems and lines of business at VA. Event-driven architecture uses events – types of actions, such as a Veteran creating a medical appointment or updating their beneficiaries – to communicate with systems that are subscribed to the stream of events. The systems that are producing the events, also known as [producers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/produce-events/), are decoupled from the systems that are consuming their events, also known as [consumers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/consume-events/).
 
-See this [brief video](https://www.youtube.com/watch?v=R6tUoxx2gVY) for a quick overview of event-driven architecture.
-
-_The following diagram illustrates how the different components of the Enterprise Event Bus come together to produce streams of events._
+_The following diagram illustrates how the different components of the Enterprise Event Bus come together to produce streams of events. Open the diagram in [Github](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Diagrams/future%20state%20whole.png)._
 
 ![future state whole](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/f0dfe62a-8509-459c-bd9a-074e0babb22b)
 
 
-# Event-driven architecture at the VA
-The Enterprise Event Bus is not intended to subsume or replace any existing VA API ecosystems, nor is it a one-stop shop or mandatory runtime environment for all VA data services; instead, it is intended to enable event-based architecture for current and future integrations between systems.
+## Role within the VA enterprise environment
 
-The focus of the Enterprise Event Bus is enterprise events rather than data events. As described above, an event is a specific instance of something that happens. An enterprise event is a business event that is of potential interest to a broad scope of consuming systems.
+We define enterprise events as those that capture significant occurrences within VA, such as when a Veteran’s benefit claim reaches certain meaningful milestones.
 
```

Comment:
I don't think this markup includes bold text for the key phrases as we do on Get Started page, where this text currently exists (https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/get-started/). I think that's Ok though...

Reply:
Agreed. Marking as resolved.